api_types.ts

```diff
@@ -62,13 +62,13 @@ interface TypeMap {
   [Type.string]: string;
   [Type.object]: object;
   [Type.boolean]: boolean;
-  [Type.date]: Date;
+  [Type.date]: string;
```

Comment:
Do you know if this affects when a date is used in an object schema property? I think that's unrelated but I forget sometimes.

Reply:
good point. let me try that.